eng/common/TestResources/New-TestResources.ps1

```diff
@@ -388,6 +397,10 @@ foreach ($templateFile in $templateFiles) {
         "$($serviceDirectoryPrefix)STORAGE_ENDPOINT_SUFFIX" = $context.Environment.StorageEndpointSuffix;
     }
 
+    foreach ($key in $EnvironmentVariables.Keys) {
+        $deploymentOutputs.Add($key, $EnvironmentVariables[$key])
```

Comment:
This will throw an exception if there are conflicts. Should we instead just output a warning if there is a conflict? That might make diagnosing a little easier.

Reply:
Originally it my my intent when I suggested it to throw - so we know there was an issue. But emitting a warning may be just as good. Hopefully nothing anybody does with this could be destructive. My fear was that some parameter gets overridden that could be very bad, but hopefully that it just limited to new resources and not existing resources...like someone deleting the wrong Key Vault accidentally.